xlators/mgmt/glusterd/src/glusterd-bitrot.c

```diff
@@ -48,11 +48,12 @@ __glusterd_handle_bitrot(rpcsvc_request_t *req)
     char msg[256] = {
         0,
     };
-    xlator_t *this = THIS;
+    xlator_t *this = NULL;
     glusterd_conf_t *conf = NULL;
 
     GF_ASSERT(req);
-
+    this = req->trans->xl;
```

Comment:
Do we know that req->trans is always not NULL?

Reply:
Hopefully yes, because all non-NULL requests created by `rpcsvc_request_init()` should have non-NULL `trans` pointer as well.